### PR TITLE
LPS-167603 Poshi Automation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-167603

cc @david-gutierrez-mesa @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/accessiblity/PageEditorAccessibility.testcase [hunk 1]
```diff
@@ -84,6 +84,35 @@ definition {
...
+				layoutName = "Test Page Name",
+				type = "content");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			Click(
+				key_tabName = "Browser",
+				locator1 = "PageEditor#FRAGMENT_SIDEBAR_TAB_BUTTON");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/accessiblity/PageEditorAccessibility.testcase:102]: Use macro
```
		PageEditor.gotoTab(tabName = "Browser");
```

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! I've updated the step you mentioned. As it's something simple I'm going to forward the PR. Thanks!

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**